processing/data_collection/gazette/spiders/sp_guaruja.py

```diff
@@ -8,30 +10,88 @@
 
 
 class SpGuaruja(BaseGazetteSpider):
+
     TERRITORY_ID = "3518701"
     name = "sp_guaruja"
     allowed_domains = ["guaruja.sp.gov.br"]
-    start_urls = ["http://www.guaruja.sp.gov.br/index.php/diario-oficial/"]
+    start_urls = ["http://www.guaruja.sp.gov.br/edicoes-diario-oficial/"]
+
+    def __init__(self, start_date=None, end_date=None, *args, **kwargs):
+        self.start_date = dt.date.today().replace(day=1)
```

Comment:
Please, set the start_date with the date if the first gazette file available.

Reply:
Done.